client/asset/dcr/dcr.go

```diff
@@ -6160,6 +6191,38 @@ func (dcr *ExchangeWallet) idUnknownTx(ctx context.Context, ltxr *ListTransactio
 		}, nil
 	}
 
+	if isMixedSplitTx(tx.MsgTx) {
+		var mixedAmount uint64
+		for _, txOut := range tx.MsgTx.TxOut {
+			_, addrs := stdscript.ExtractAddrs(scriptVersion, txOut.PkScript, dcr.chainParams)
+			if err != nil {
+				dcr.log.Errorf("ExtractAddrs error: %w", err)
+				continue
+			}
+			if len(addrs) != 1 { // sanity check
+				continue
+			}
+
+			addr := addrs[0]
+			owns, err := dcr.wallet.WalletOwnsAddress(ctx, addr)
+			if err != nil {
+				dcr.log.Errorf("walletOwnsAddress error: %w", err)
+				continue
+			}
+
+			if owns {
+				mixedAmount += uint64(txOut.Value)
+			}
```

Comment:
Wouldn't this also count the value of a change output? How about changing the `isMixedSplitTx` signature to also return the `ticketOutAmt` as it does in dcrdata. Then you could reuse the same block above where it checks `if txOut.Value == mixDenom { `.

Reply:
Ok, I wasn't sure what we wanted to show. If just ticket amounts can do that.

Reply:
Only adding tickets for mixed amount https://github.com/decred/dcrdex/compare/909e4b5a918c4b1ad7797f7fd5cce6128ea4b6a6..74578b75da4ced4016b297655266295a75bdb374

Reply:
Looks like the inputs of these transactions are all the size of a mix output or a ticket redemption right? And the outputs are either the size of a ticket purchase or a change, is this correct? If so, then I think it makes sense to display the ticket amounts.